cmd/kops/integration_test.go

```diff
@@ -33,8 +33,13 @@ import (
 
 	"golang.org/x/crypto/ssh"
 
+	"encoding/base64"
```

Comment:
ok to sort these, standard libraries at top and then alphabetical?

Reply:
Fixed them up a bit.  Is there a tool you've found other than goimports?  (`goimports` is good, but doesn't change the newline separated blocks)

Reply:
Cheers! I'm using goimports too, so I just check manually when I remember to. :)